dialoguekit/platforms/platform_terminal.py

```diff
@@ -0,0 +1,46 @@
+"""The Platform facilitates displaying of the conversation."""
```

Comment:
Nit: maybe you can mention here that it is the terminal.

Reply:
Done.